wasp/draw565.py

```diff
@@ -310,8 +310,18 @@ def string(self, s, x, y, width=None, right=False):
         font = self._font
         bg = self._bgfg >> 16
 
+        (w, h) = _bounding_box(s, font)
         if width:
+            wmax = min(width, 240 - x)
+            width = wmax
+        else:
+            wmax = 240 - x
+        while w > wmax:
```

Comment:
Maybe it is needed to change to
```
while w > wmax - 1
```
otherwise it does not work if the string has exactly the required width. Best would probably to change the following lines, but I don't have time / I don't understand them fully